roles/hosted_engine_setup/tasks/initial_clean.yml

```diff
@@ -56,12 +60,22 @@
         src: /usr/share/abrt/conf.d/plugins/vmcore.conf,
         dest: /etc/abrt/plugins/vmcore.conf
       }
-    when: abrt_exists.stat.exists
+    when: abrt_vdsm_config.stat.exists and abrt_installed_config.stat.exists
```

Comment:
Please use "more readable" :-) syntax:
```
    when:
      - abrt_vdsm_config.stat.exists
      - abrt_installed_config.stat.exists
```

Reply:
Done, hope the one with `not` will work too